dd-trace-api/src/main/java/datadog/trace/api/Config.java

```diff
@@ -117,6 +117,9 @@
   public static final String LANGUAGE_TAG_KEY = "language";
   public static final String LANGUAGE_TAG_VALUE = "jvm";
 
+  public static final String MAX_SPANS_PER_TRACE = "maxSpansPerTrace";
```

Comment:
The pattern has been dot-separating terms: `max.spans.per.trace`.  This allows configuring w/ `SIGNALFX_MAX_SPAN_PER_TRACE` env var.   Would you mind adding to the configuration table in the main readme as well?

Reply:
Will do both of these once I get tests passing 😄